index.html

```diff
@@ -70,5 +70,32 @@
         </div>
       </div>
     </div>
+    <div class="about">
+      <div class="meet_team">Meet your Pharmacy Team</div>
+      <div class="meet1">
+        <img class="meet_pic1"
+        src="https://scontent.fxds1-1.fna.fbcdn.net/v/t1.6435-9/93763348_104287811255407_4968196513340588032_n.jpg?_nc_cat=103&ccb=1-7&_nc_sid=09cbfe&_nc_ohc=lrLKb4P3uc0AX_URdpo&_nc_ht=scontent.fxds1-1.fna&oh=00_AT9DUl_nbRvxqc8rcHSVPaJs0z_i3RPcCglCTgbbGT0wgg&oe=62C5B06E"
+        alt = "Pharmacist Tu Chu">
+        <div class="meet_person1"> Tu Chu</div>
+        <div class="title1"> Head Pharmacist</div>
```

Comment:
I'd change the class to something a bit more specific so the class doesn't accidentally overlap with anything else since it's a bit too generic. Maybe like `person-title`?